tests/tests/test_database_backend.py

```diff
@@ -232,6 +232,42 @@ def test_missing_task_path(self) -> None:
         with self.assertRaises(ImportError):
             _ = db_task_result.task
 
+    def test_task_name(self) -> None:
+        db_task_result = DBTaskResult.objects.create(
+            args_kwargs={"args": [], "kwargs": {}},
+            task_path="tests.tasks.noop_task",
```

Comment:
Suggestion: Given these tests are incredibly similar, it'd be better to use `subTest` to just add cases and expected values.

Reply:
Updated in a965773ed51df64cd409ea0ec541603e9f35c3d7